odoo/models.py

```diff
@@ -261,34 +261,74 @@ class BaseModel(MetaModel('DummyModel', (object,), {'_register': False})):
-    attribute may be set to False.
+    To create a class that should not be instantiated,
+    the :attr:`~odoo.models.BaseModel._register` attribute may be set to False.
     """
-    _auto = False               # don't create any database backend
-    _register = False           # not visible in ORM registry
-    _abstract = True            # whether model is abstract
-    _transient = False          # whether model is transient
 
-    _name = None                # the model name
-    _description = None         # the model's informal name
-    _custom = False             # should be True for custom models only
+    _auto = False
+    """Whether a database table should be created (default: ``True``)
+    If set to ``False``, override :meth:`~odoo.models.BaseModel.init`
+    to create the database table.
 
-    _inherit = None             # Python-inherited models ('model' or ['model'])
-    _inherits = {}              # inherited models {'parent_model': 'm2o_field'}
+    .. tip:: To create a model without any table, inherit
+            from :class:`~odoo.models.AbstractModel`.
+    """
+    _register = False           #: not visible in ORM registry
+    _abstract = True            #: whether model is abstract
+    _transient = False          #: whether model is transient
+
+    _name = None                #: the model name (in dot-notation, module namespace)
+    _description = None         #: the model's informal name
+    _custom = False             #: should be True for custom models only
+
+    _inherit = None
+    """Python-inherited models ('model' or ['model'])
+
+    * If :attr:`._name` is set, names of parent models to inherit from.
+        Can be a ``str`` if inheriting from a single parent
+    * If :attr:`._name` is unset, name of a single model to extend in-place
+    """
+    _inherits = {}
+    """dictionary {'parent_model': 'm2o_field'} mapping the _name of the parent business
+    objects to the names of the corresponding foreign key fields to use::
 
-    _table = None               # SQL table name used by model
-    _sequence = None            # SQL sequence to use for ID field
-    _sql_constraints = []       # SQL constraints [(name, sql_def, message)]
+      _inherits = {
+          'a.model': 'a_field_id',
+          'b.model': 'b_field_id'
+      }
 
-    _rec_name = None            # field to use for labeling records
-    _order = 'id'               # default order for searching results
-    _parent_name = 'parent_id'  # the many2one field used as parent field
-    _parent_store = False       # set to True to compute parent_path field
-    _date_name = 'date'         # field to use for default calendar view
-    _fold_name = 'fold'         # field to determine folded groups in kanban views
+    implements composition-based inheritance: the new model exposes all
+    the fields of the inherited models but stores none of them:
+    the values themselves remain stored on the linked record.
+
+    .. ??? .. warning::
```

Comment:
missing the end of your warning i believe

Reply:
In fact, the warning was like that in the doc, and as it is incomplete, i really dunno what it wanted to say...
That's why there is a .. before, which comments the following block in .rst files.
If you have a clue on what was the objective of this warning, i'm interested :D